next.config.js

```diff
@@ -118,6 +118,7 @@ module.exports = withPlugins([[withBundleAnalyzer], [withSourceMaps]], {
     VERCEL_URL: process.env.VERCEL_URL,
     SITE_IMAGERY_API_URL: SITE_IMAGERY_API_URL,
     WIDGET_URL: process.env.WIDGET_URL,
+    TOPBAR_API_URL: process.env.TOPBAR_API_URL,
```

Comment:
Please add values to https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.production and https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.development and https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.local.example so we have a default value and do not need to set the env variable in all of our deployments.

Reply:
Yes, but this url won't work, will have to find some other solution

Reply:
https://30c3f8e4.planet-ads.pages.dev/topbar/ten_I9TW3ncG/messages.json
gives CORS error when fetched in the webapp

Reply:
@sarveshpro isn't there a possibility to add a header, e.g. `access-control-allow-origin: *`, to allow fetching it?